# Inputs
string table_name
string entry_name
bool[] value
---
# Outputs
bool success
time last_valid